lumigator/python/mzai/sdk/lumigator_sdk/jobs.py

```diff
@@ -130,16 +134,18 @@ def create_job(self, type: JobType, request: JobEvalCreate | JobInferenceCreate)
 
         Args:
             type(JobType): The kind of job to create. It can be either
-                EVALUATION or INFERENCE.
+                EVALUATION_LITE, ANNOTATION or INFERENCE.
```

Comment:
❓ Is `EVALUATION` not supported now (it looks like it is in the code below)? Sorry if this is a dumb question, I just haven't followed these developments.

Suggestion: make the job types lexicographical in their ordering.

```suggestion
                ANNOTATION, EVALUATION_LITE or INFERENCE.
```

Reply:
Yes, standalone evaluation is still supported. I will use lexicographical order 👍